### PR TITLE
Specification of how to serialise datamodels

### PR DESCRIPTION
**Pull request by jesper-friis**
Added an example specifying how instances can be serialised: https://github.com/emmo-repo/datamodel/blob/serialisation/docs/serialisation.md

### CODE REVIEW

**Comment by francescalb** [comment]
I think it would be very useful to have an example to better evaluate which is the best alternative. My immediate respons is that I think speciffic data models (entitities) and instances of these should conceptually both be individuals (alternative 2, as I understand), but I do not properly grasp how these two alternatives will be implemented (differently).

**Comment by jesper-friis** [comment]
> I think it would be very useful to have an example to better evaluate which is the best alternative. My immediate respons is that I think speciffic data models (entitities) and instances of these should conceptually both be individuals (alternative 2, as I understand), but I do not properly grasp how these two alternatives will be implemented (differently).

I think we already have agreed on going for individuals. Instead of spending more time on the alternatives, added docs/serialisation.md with figures and turtle examples about how to use the ontology.

File: docs/serialisation.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    dm:instanceOf <http://onto-ns.com/meta/0.1/Person> ;
+    dm:hasDimension :dim1 ;
+    dm:hasProperty :name1 , :age1 , :skills1 .
+
+:dim1 rdf:type dm:DimensionValue ;
+    dm:instanceOf person:nskills ;
+    dm:hasValue "4"^^xsd:integer .
+
+:name1 rdf:type dm:PropertyValue ;
+    dm:hasValue "Shelock Homes"^^xsd:string .
```
Comment (francescalb) [inline docs/serialisation.md]:
```suggestion
    dm:hasValue "Sherlock Holmes"^^xsd:string .
```

**Review by francescalb — APPROVED** [review]
This looks OK to me.

**Pull request merged**